fem/fe/fe_base.hpp

```diff
@@ -217,9 +217,11 @@ class FunctionSpace
 public:
    enum
    {
-      Pk, ///< Polynomials of order k
-      Qk, ///< Tensor products of polynomials of order k
-      rQk ///< Refined tensor products of polynomials of order k
+      Pk,  ///< Polynomials of order k
+      Qk,  ///< Tensor products of polynomials of order k
+      rQk, ///< Refined tensor products of polynomials of order k
+      RBF, ///< Radial basis functions
+      RK,  ///< Reproducing kernels
```

Comment:
The only place I see `FunctionSpace` used is in the order functions for the `IsoparametricTransformation`. It turns out that I have been using `FunctionSpace::Qk` for both `RBFFiniteElement` and `RKFiniteElement`. If I add `FunctionSpace::RBF` to the order functions in `IsoparametricTransformation`, I get an error there in ex18 when it tries to do a transformation from `FunctionSpace::Qk` to `FunctionSpace::RBF`. What would you guys recommend doing here? If it is just a matter of choosing the correct integration order for different things, then it's possible that continuing to use `FunctionSpace::Qk` might be the easiest choice. If not, then I will need to address the compatibility check (`FElem->Space() == fe->Space()`) in `IsoparametricTransformation::OrderGrad`. If we do want a separate function space for RBF, I would suggest that we eliminate `FunctionSpace::RK`, since it should have similar properties under integration to RBF.

Reply:
I think that's fine if you don't need customized quad rules for the RBF. For example the `NURBSFiniteElements` also use `Qk`.